### ISSUE TITLE
BUG: Decentralized Supply system failed to execute

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#1303]
```Executing: DecentralizedBuildings C:\Users\Gabriel\Documents\GitHub\cea-reference-case\reference-case-WTP\03_MIX_medium_density\baseline true false false false false false
Start Time: Mon May 14 15:50:27 2018
Running script DecentralizedBuildings...
Executing: C:\Users\Gabriel\Anaconda2\envs\cea\python.exe -u -m cea.interfaces.cli.cli decentralized --scenario C:\Users\Gabriel\Documents\GitHub\cea-reference-case\reference-case-WTP\03_MIX_medium_density\baseline --scuflag false --ahuscuflag false --aruscuflag false --aruflag false --ahuflag true --ahuaruflag false
B001
1.77913269892 seconds process time for the Substation Routine.
B001
3.05989384378 seconds process time for the Substation Routine.
B001
1.9117211275 seconds process time for the Substation Routine.
B001
3.3478572744 seconds process time for the Substation Routine.
B001
2.06216648403 seconds process time for the Substation Routine.
B001
3.14566595616 seconds process time for the Substation Routine.
B001
3.15573639269 seconds process time for the Substation Routine.
B001
B002
1.7859141524 seconds process time for the Substation Routine.
B002
3.19350481459 seconds process time for the Substation Routine.
B002
2.16128051171 seconds process time for the Substation Routine.
B002
3.27151742133 seconds process time for the Substation Routine.
B002
2.02946518926 seconds process time for the Substation Routine.
B002
3.21545409956 seconds process time for the Substation Routine.
B002
3.2507536873 seconds process time for the Substation Routine.
B002
B003
1.80699661037 seconds process time for the Substation Routine.
B003
3.13612976513 seconds process time for the Substation Routine.
B003
1.99167269896 seconds process time for the Substation Routine.
B003
3.16125024387 seconds process time for the Substation Routine.
B003
1.98932748885 seconds process time for the Substation Routine.
B003
3.15461356546 seconds process time for the Substation Routine.
B003
3.2793713772 seconds process time for the Substation Routine.
B003
B004
1.79421191495 seconds process time for the Substation Routine.
B004
3.08744997146 seconds process time for the Substation Routine.
B004
1.97817288039 seconds process time for the Substation Routine.
B004
3.0970288292 seconds process time for the Substation Routine.
B004
1.98834360191 seconds process time for the Substation Routine.
B004
3.10888433858 seconds process time for the Substation Routine.
B004
3.18257156246 seconds process time for the Substation Routine.
B004
B005
1.80242653213 seconds process time for the Substation Routine.
B005
3.11564354703 seconds process time for the Substation Routine.
B005
1.9696391746 seconds process time for the Substation Routine.
B005
3.09854222089 seconds process time for the Substation Routine.
B005
1.98257411517 seconds process time for the Substation Routine.
B005
3.10564969176 seconds process time for the Substation Routine.
B005
3.18736044432 seconds process time for the Substation Routine.
B005
B006
1.0870448201 seconds process time for the Substation Routine.
B006
1.37807806548 seconds process time for the Substation Routine.
B006
0.597748146095 seconds process time for the Substation Routine.
B006
1.40678327684 seconds process time for the Substation Routine.
B006
1.26569286901 seconds process time for the Substation Routine.
B006
1.39675514234 seconds process time for the Substation Routine.
B006
1.39625444673 seconds process time for the Substation Routine.
B006
B007
1.83146870105 seconds process time for the Substation Routine.
B007
3.17695961432 seconds process time for the Substation Routine.
B007
2.07785506813 seconds process time for the Substation Routine.
B007
3.3213484867 seconds process time for the Substation Routine.
B007
1.98395184862 seconds process time for the Substation Routine.
B007
3.27822521092 seconds process time for the Substation Routine.
B007
3.27123844671 seconds process time for the Substation Routine.
B007
B008
1.80647075234 seconds process time for the Substation Routine.
B008
3.61209801673 seconds process time for the Substation Routine.
B008
2.00340604299 seconds process time for the Substation Routine.
B008
3.14886267703 seconds process time for the Substation Routine.
B008
1.99079420243 seconds process time for the Substation Routine.
B008
3.24139144514 seconds process time for the Substation Routine.
B008
3.23313087921 seconds process time for the Substation Routine.
B008
B009
1.83838289539 seconds process time for the Substation Routine.
B009
3.21237152125 seconds process time for the Substation Routine.
B009
2.01739014691 seconds process time for the Substation Routine.
B009
3.15849258894 seconds process time for the Substation Routine.
B009
2.01346955074 seconds process time for the Substation Routine.
B009
3.238278599 seconds process time for the Substation Routine.
B009
3.30603989155 seconds process time for the Substation Routine.
B009
B010
1.81221981741 seconds process time for the Substation Routine.
B010
3.13445518811 seconds process time for the Substation Routine.
B010
2.00386115452 seconds process time for the Substation Routine.
B010
3.29482073602 seconds process time for the Substation Routine.
B010
1.99929800506 seconds process time for the Substation Routine.
B010
3.17370418115 seconds process time for the Substation Routine.
B010
3.24450392661 seconds process time for the Substation Routine.
B010
B011
1.80253921599 seconds process time for the Substation Routine.
B011
3.10259118185 seconds process time for the Substation Routine.
B011
2.03842847948 seconds process time for the Substation Routine.
B011
3.1357107562 seconds process time for the Substation Routine.
B011
1.97734434402 seconds process time for the Substation Routine.
B011
3.11308318002 seconds process time for the Substation Routine.
B011
3.19066839035 seconds process time for the Substation Routine.
B011
B012
1.83387955225 seconds process time for the Substation Routine.
B012
3.18392705087 seconds process time for the Substation Routine.
B012
1.94304870022 seconds process time for the Substation Routine.
B012
3.20926159249 seconds process time for the Substation Routine.
B012
2.00561632421 seconds process time for the Substation Routine.
B012
3.18257302115 seconds process time for the Substation Routine.
B012
3.30982811152 seconds process time for the Substation Routine.
B012
B013
1.78691080288 seconds process time for the Substation Routine.
B013
3.18898688455 seconds process time for the Substation Routine.
B013
1.92793775766 seconds process time for the Substation Routine.
B013
3.13070562344 seconds process time for the Substation Routine.
B013
1.98216312904 seconds process time for the Substation Routine.
B013
3.1217919287 seconds process time for the Substation Routine.
B013
3.19450328844 seconds process time for the Substation Routine.
B013
B014
1.76273154437 seconds process time for the Substation Routine.
B014
3.13747212533 seconds process time for the Substation Routine.
B014
1.94928752072 seconds process time for the Substation Routine.
B014
3.14221761114 seconds process time for the Substation Routine.
B014
1.94935097377 seconds process time for the Substation Routine.
B014
3.15485133206 seconds process time for the Substation Routine.
B014
3.1834124977 seconds process time for the Substation Routine.
B014
B015
1.78650893357 seconds process time for the Substation Routine.
B015
3.10336282928 seconds process time for the Substation Routine.
B015
1.95506247755 seconds process time for the Substation Routine.
B015
3.10113285573 seconds process time for the Substation Routine.
B015
1.97427963467 seconds process time for the Substation Routine.
B015
3.13685655782 seconds process time for the Substation Routine.
B015
3.17503049575 seconds process time for the Substation Routine.
B015
B016
1.78031715584 seconds process time for the Substation Routine.
B016
3.10024597173 seconds process time for the Substation Routine.
B016
1.91336178996 seconds process time for the Substation Routine.
B016
3.16854515651 seconds process time for the Substation Routine.
B016
1.95752839433 seconds process time for the Substation Routine.
B016
3.10952980926 seconds process time for the Substation Routine.
B016
3.19644881728 seconds process time for the Substation Routine.
B016
B017
1.77328480755 seconds process time for the Substation Routine.
B017
3.08524734837 seconds process time for the Substation Routine.
B017
1.94363582326 seconds process time for the Substation Routine.
B017
3.15412563339 seconds process time for the Substation Routine.
B017
1.93156005156 seconds process time for the Substation Routine.
B017
3.15653356721 seconds process time for the Substation Routine.
B017
3.14423206312 seconds process time for the Substation Routine.
B017
B018
1.06843119629 seconds process time for the Substation Routine.
B018
1.32732838959 seconds process time for the Substation Routine.
B018
0.575444763939 seconds process time for the Substation Routine.
B018
1.38943105589 seconds process time for the Substation Routine.
B018
1.23539403797 seconds process time for the Substation Routine.
B018
1.35459277912 seconds process time for the Substation Routine.
B018
1.43911953424 seconds process time for the Substation Routine.
B018
B019
1.74871571393 seconds process time for the Substation Routine.
B019
3.07608421751 seconds process time for the Substation Routine.
B019
1.92491024493 seconds process time for the Substation Routine.
B019
3.06617678968 seconds process time for the Substation Routine.
B019
2.07038365391 seconds process time for the Substation Routine.
B019
3.09783621455 seconds process time for the Substation Routine.
B019
3.15973247611 seconds process time for the Substation Routine.
B019
B020
1.76263745881 seconds process time for the Substation Routine.
B020
3.11674011783 seconds process time for the Substation Routine.
B020
1.91020700646 seconds process time for the Substation Routine.
B020
3.21983783005 seconds process time for the Substation Routine.
B020
1.96765243774 seconds process time for the Substation Routine.
B020
3.16636769583 seconds process time for the Substation Routine.
B020
3.21193865476 seconds process time for the Substation Routine.
B020
B021
1.82309800396 seconds process time for the Substation Routine.
B021
3.26281414274 seconds process time for the Substation Routine.
B021
2.22440863764 seconds process time for the Substation Routine.
B021
3.34369088883 seconds process time for the Substation Routine.
B021
2.08454900016 seconds process time for the Substation Routine.
B021
3.36174255202 seconds process time for the Substation Routine.
B021
3.2866440448 seconds process time for the Substation Routine.
B021
B022
1.08414129608 seconds process time for the Substation Routine.
B022
1.31829180015 seconds process time for the Substation Routine.
B022
0.613968422991 seconds process time for the Substation Routine.
B022
1.41136757732 seconds process time for the Substation Routine.
B022
1.26250161823 seconds process time for the Substation Routine.
B022
1.38510275565 seconds process time for the Substation Routine.
B022
1.41971347666 seconds process time for the Substation Routine.
B022
c:\users\gabriel\documents\github\ceaforarcgis\cea\technologies\cooling_tower.py:51: RuntimeWarning: invalid value encountered in double_scalars
  qpartload = qhotdot_W / Qdesign_W
c:\users\gabriel\documents\github\ceaforarcgis\cea\technologies\cooling_tower.py:55: RuntimeWarning: invalid value encountered in double_scalars
  qpartload + 0.0026
Traceback (most recent call last):
  File "C:\Users\Gabriel\Anaconda2\envs\cea\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Users\Gabriel\Anaconda2\envs\cea\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "c:\users\gabriel\documents\github\ceaforarcgis\cea\interfaces\cli\cli.py", line 79, in <module>
    main(cea.config.Configuration())
  File "c:\users\gabriel\documents\github\ceaforarcgis\cea\interfaces\cli\cli.py", line 33, in main
    script_module.main(config)
  File "c:\users\gabriel\documents\github\ceaforarcgis\cea\optimization\preprocessing\disconnected_building_main.py", line 48, in main
    disconnected_building_main(locator=locator, building_names=building_names, config=config, prices=prices)
  File "c:\users\gabriel\documents\github\ceaforarcgis\cea\optimization\preprocessing\disconnected_building_main.py", line 33, in disconnected_building_main
    disconnected_buildings_cooling.disconnected_buildings_cooling_main(locator, building_names, config, prices)
  File "c:\users\gabriel\documents\github\ceaforarcgis\cea\optimization\preprocessing\disconnected_buildings_cooling.py", line 1138, in disconnected_buildings_cooling_main
    hour] > 0 else 0
  File "c:\users\gabriel\documents\github\ceaforarcgis\cea\technologies\boiler.py", line 166, in calc_Cop_boiler
    boiler_eff = (eff_score * eff_of_T_return(T_return_C)) / 100.0
  File "C:\Users\Gabriel\Anaconda2\envs\cea\lib\site-packages\scipy\interpolate\polyint.py", line 79, in __call__
    y = self._evaluate(x)
  File "C:\Users\Gabriel\Anaconda2\envs\cea\lib\site-packages\scipy\interpolate\interpolate.py", line 610, in _evaluate
    below_bounds, above_bounds = self._check_bounds(x_new)
  File "C:\Users\Gabriel\Anaconda2\envs\cea\lib\site-packages\scipy\interpolate\interpolate.py", line 639, in _check_bounds
    raise ValueError("A value in x_new is below the interpolation "
ValueError: A value in x_new is below the interpolation range.


Traceback (most recent call last):
  File "C:\Users\Gabriel\AppData\Roaming\ESRI\Desktop10.5\ArcToolbox\My Toolboxes\cea\interfaces\arcgis\arcgishelper.py", line 68, in execute
    run_cli(self.cea_tool, **kwargs)
  File "C:\Users\Gabriel\AppData\Roaming\ESRI\Desktop10.5\ArcToolbox\My Toolboxes\cea\interfaces\arcgis\arcgishelper.py", line 155, in run_cli
    raise Exception('Tool did not run successfully')
Exception: Tool did not run successfully

Failed to execute (DecentralizedBuildings).
Failed at Mon May 14 18:57:04 2018 (Elapsed Time: 3 hours 6 minutes 36 seconds)```

**Comment by gabriel-happle** [comment]
Assigning this to you @BhargavaKrishnaSreepathi .

**Comment by gabriel-happle** [comment]
duplicate of #1300

**Comment by BhargavaKrishnaSreepathi** [comment]
fixed this in PR of cost-analysis-plots #1358
